### DISCUSSION TITLE
Deploy multiple lambdas

### DISCUSSION THREAD

**Discussion by ootsun** [quarkusio/quarkus#36225] [Q&A] [answered]
Hello,

I followed this guide : https://quarkus.io/guides/funqy-aws-lambda
It defines multiple functions and explains how to deploy one of them.

How can we deploy all of them? It seems like we can only build 1 .zip at a time.
If there are 100's of endpoints in my application, does it mean I'll have to build and deploy my app 100's time to have all of my functions available on aws?

[a comment by an automated account is not shown]

**Comment by patriot1burke** [comment, marked as answer]
If you're using funqy or raw AWS Lambda, the answer is you have to deploy your app 100's of times.  That's just the way they work... 

Alternatively, you could create a REST app with 100's of endpoints and deploy it to lambda and invoke on them via HTTP:

https://quarkus.io/guides/aws-lambda-http

**Reply by ootsun** [reply]
Thanks for your help. I'll use this then :) 
For what kind of tasks do people use Funqy or raw AWS Lambda then? This limitation seems... limiting!
